core/build.gradle

```diff
@@ -57,7 +61,8 @@ sourceSets {
             compileClasspath += main.output + test.output
             runtimeClasspath += main.output + test.output
             srcDir file('src/integration-test/java')
-            include '**/com/scalar/db/storage/jdbc/*.java'
+            include '**/com/scalar/db/common/*.java'
+            include '**/com/scalar/db/storage/jdbc/JdbcTestUtils.java'
```

Comment:
This is a bit unrelated bug fix to prevent this integration test task from executing JDBC integration tests.